src/Camspiers/LoggerBridge/BacktraceReporter/BasicBacktraceReporter.php

```diff
@@ -33,14 +33,14 @@ public function getBacktraceLimit()
 
     /**
      * Returns a basic backtrace
-     * @param  \Exception $exception
+     * @param  mixed      $exception
      * @return array
      */
-    public function getBacktrace(\Exception $exception = null)
+    public function getBacktrace($exception = null)
     {
         $skipLimit = false;
 
-        if ($exception instanceof \Exception) {
+        if ($exception instanceof \Exception || $exception instanceof \Throwable) {
```

Comment:
The `\Throwable` interface also include a `getTrace()` method, so it makes sense to make use of it here: http://php.net/manual/en/throwable.gettrace.php. It’s still compatible with PHP 5 because PHP doesn’t check whether `\Throwable` actually exists, it just treats it as a string.